core/templates/pages/preferences-page/preferences-page.component.css

```diff
@@ -71,3 +71,11 @@
 .oppia-form-text {
   font-size: smaller;
 }
+.focus-class:focus-visible {
+  outline: 2px solid #0060df;
+  text-decoration: underline;
+}
+a.delete-btn, a.delete-btn:hover, a.delete-btn:visited,a.delete-btn:active {
+  color: #fff;
+  text-decoration: none;
+}
```

Comment:
Please make sure these are prefixed with the preferences page class so that they are not applied in different places.